frontend/src/types/CamperTypes.ts

```diff
@@ -1,3 +1,11 @@
+export type EmergencyContact = {
```

Comment:
nit: is there a reason why this is "EmergencyContact" and not just "Contact"?

Reply:
we refer to contacts as Emergency Contacts throughout our product- or at least that's the terminology we used to begin with so I wanted to keep it consistent